staging/src/k8s.io/component-base/go.mod

```diff
@@ -13,16 +13,20 @@ require (
 	github.com/spf13/pflag v1.0.3
 	github.com/stretchr/testify v1.3.0
 	k8s.io/apimachinery v0.0.0
+	k8s.io/client-go v0.0.0
```

Comment:
there is a rules file in staging/ and some import restriction file. Both need to be changed to include/allow client-go here.

Reply:
done.